homeassistant/components/knx/__init__.py

```diff
@@ -148,6 +148,12 @@ async def async_setup(hass, config):
             discovery.async_load_platform(hass, platform.value, DOMAIN, {}, config)
         )
 
+    if len(hass.data[DATA_KNX].xknx.devices) == 0:
+        _LOGGER.warning(
+            "No KNX devices are configured. Please read "
+            "https://home-assistant.io/blog/2020/09/17/release-115/#breaking-changes."
```

Comment:
@MartinHjelmare Is this the correct link? The current rc points to the 7th September (https://rc.home-assistant.io/blog/2020/09/07/release-115/) and states the release will be at the 17th september.

A manually adapted the link to the 17th September. Is this correct?

Reply:
I think it should be:
```suggestion
            "https://www.home-assistant.io/blog/2020/09/17/release-115/#breaking-changes"
```